src/main/src/main/java/org/geoserver/security/GeoServerSecurityManager.java

```diff
@@ -2625,13 +2626,15 @@ boolean migrateFrom22(boolean migratedFrom21) throws Exception{
             
         // set redirect url after successful logout
         if (migratedFrom21== false)
```

Comment:
On 02/12/15 00:35, NielsCharlier wrote:
> Personally, I dislike "==false" instead of using "!". Even worse is "==true" ;)

Good point. I like your previous conversions to "!".

But not like this amusing Linux kernel patch example (literal 
misinterpretation of code checker), seen on Reddit:
https://lkml.org/lkml/2015/11/19/454

-- 
Ben Caradoc-Davies <ben@transient.nz>
Director
Transient Software Limited <http://transient.nz/>
New Zealand